seed.js

```diff
@@ -0,0 +1,7 @@
+require('dotenv').config();
+require('./lib/utils/connect')();
+
+const seedData = require('./db/seed');
+
+seedData({ studiosToCreate: 10 })
+  .then(() => console.log('done'));
```

Comment:
It seems like you're already doing this in your seed.js file inside your db folder so I'm not sure what the point of this file is.